app/views/issues/show.html.erb

```diff
@@ -0,0 +1,29 @@
+<div class="card" style="border-left: 4px solid green">
+  <div class="card-body">
+    <h5 class="card-title"><%= @issue.title %><span class="float-right"><%= render 'show_badges', issue: @issue %></span></h5>
+    <hr>
+    <h6 class="card-subtitle mb-2 text-muted">Creator: <%= @issue.creator.email %></h6>
+    <% if @issue.assignee %>
```

Comment:
change to @issue.assignee_id and explain the reason over here

Reply:
It is to ensure that the assignee will be displayed if assignee exists else 'Not assigned to anyone' will be displayed at assignee's position